docs: rework mdbook (#1036)
coreos/clair: Documentation/howto/api.md

# API Definition

> ClairV4 provides its API defintion via an OpenAPI specification. You can view our OpenAPI spec [here](https://raw.githubusercontent.com/quay/clair/development-4.0/openapi.yaml)

Reviewer: ```suggestion
Clair provides its API definition via an OpenAPI specification. You can view our OpenAPI spec [here](https://raw.githubusercontent.com/quay/clair/development-4.0/openapi.yaml)
```

The OpenAPI spec can be used in a variety of ways.
* Generating http clients for your application

> * Validating data returned from ClairV4

Reviewer: ```suggestion
* Validating data returned from Clair
```

> * Importing into a rest client such as [PostMan](https://learning.postman.com/docs/integrations/available-integrations/working-with-openAPI/)

Reviewer: ```suggestion
* Importing into a rest client such as [Postman](https://learning.postman.com/docs/integrations/available-integrations/working-with-openAPI/)
```

Postman isn't capped like that.

* API documentation via [Swagger Editor](https://petstore.swagger.io/#/)

> See [Testing ClairV4](./testing.md) to learn how the local dev tooling starts a local swagger editor. This is handy for making changes to the spec in real time.

Reviewer: ```suggestion
See [Testing Clair](./testing.md) to learn how the local dev tooling starts a local swagger editor. This is handy for making changes to the spec in real time.
```

> See [Api Reference](./reference/api.md) for a markdown rendered api reference.

Reviewer: ```suggestion
See [API Reference](./reference/api.md) for a markdown rendered API reference.
```
